versioned_docs/version-0.x.0/kubeslice-cli/kubeslice-cli-overview.mdx

```diff
@@ -0,0 +1,20 @@
+# KubeSlice CLI Overview
```

Comment:
Change the title to `kubeslice-cli Overview`

Reply:
done